packages/shorebird_cli/test/src/commands/collaborators/add_collaborators_command_test.dart

```diff
@@ -82,12 +82,6 @@ void main() {
     test('returns ExitCode.success when user aborts', () async {
       when(() => logger.confirm(any())).thenReturn(false);
       expect(await command.run(), ExitCode.success.code);
-      verifyNever(
```

Comment:
This should probably have been verifying the `createCollaborator` was never called instead

Reply:
Ah! Fixed.